<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ROSGroundControl</class>
 <widget class="QMainWindow" name="ROSGroundControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ROSGroundControl</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="upCommand">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="rightCommand">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>70</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="leftCommand">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>70</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="downCommand">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>10</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QOpenGLWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="laserMeasurement">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>260</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Dist:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sensorTrigger">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>140</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Sensor Btn</string>
    </property>
   </widget>
   <widget class="QPushButton" name="chartCloseButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>200</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Close Plot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>370</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cameraMode_1">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>320</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="cameraMode_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>320</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="cameraMode_3">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>320</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cameraModesLabel">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>300</y>
      <width>141</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Camera Modes</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="distanceText">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>260</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <zorder>upCommand</zorder>
   <zorder>rightCommand</zorder>
   <zorder>leftCommand</zorder>
   <zorder>downCommand</zorder>
   <zorder>openGLWidget</zorder>
   <zorder>sensorTrigger</zorder>
   <zorder>chartCloseButton</zorder>
   <zorder>quitButton</zorder>
   <zorder>cameraMode_1</zorder>
   <zorder>cameraMode_2</zorder>
   <zorder>cameraMode_3</zorder>
   <zorder>cameraModesLabel</zorder>
   <zorder>distanceText</zorder>
   <zorder>laserMeasurement</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
